chia/full_node/bitcoin_fee_estimator.py

```diff
@@ -71,6 +62,9 @@ def get_tracker(self) -> FeeTracker:
         """
         return self.tracker
 
+    def get_mempool_info(self) -> FeeMempoolInfo:
+        return self.last_mempool_info if self.last_mempool_info else EmptyFeeMempoolInfo
```

Comment:
I don't think `self.last_mempool_info` is ever "falsely", is it?
It would be good to have mypy type annotations for the members, but as far as I can tell it's always initialized.

(I would expect mypy to alert on this, if it had a non-`Optional` type)

Reply:
You know, it used to be, when we relied on it being initialized with the first real mempoolInfo being passed in.

But now, since we init it in the constructor, you are correct. Done.